In [1]:
import numpy as np

### 1) Известно, что генеральная совокупность распределена нормально со средним квадратическим отклонением, равным 16. Найти доверительный интервал для оценки математического ожидания a с надежностью 0.95, если выборочная средняя M = 80, а объем выборки n = 256.

In [2]:
X = 80
Z = 1.96
σ = 16
n = 256

Левая граница:

In [3]:
a = X - Z*σ/np.sqrt(n)
a

78.04

Правая граница:

In [4]:
b = X + Z*σ/np.sqrt(n)
b

81.96

Ответ: [78.04, 81.96]

### 2)В результате 10 независимых измерений некоторой величины X, выполненных с одинаковой точностью, получены опытные данные: 6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1 Предполагая, что результаты измерений подчинены нормальному закону распределения вероятностей, оценить истинное значение величины X при помощи доверительного интервала, покрывающего это значение с доверительной вероятностью 0,95.

In [5]:
X = np.array([6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1])
mean_X = X.mean()
std_X = X.std(ddof=1)
t = 2.2622

In [6]:
a = mean_X - t*std_X/np.sqrt(len(X))
a

6.267509744706053

In [7]:
b = mean_X + t*std_X/np.sqrt(len(X))
b

6.912490255293949

Ответ: [6.27, 6.91]

### 3) Утверждается, что шарики для подшипников, изготовленные автоматическим станком, имеют средний диаметр 17 мм. Используя односторонний критерий с α=0,05, проверить эту гипотезу, если в выборке из n=100 шариков средний диаметр оказался равным 17.5 мм, а дисперсия известна и равна 4 кв.мм.

H0: d = 17mm, m=m0
H1: d = 17.5mm, m>m0

Критерий: Zтабличное = 1.645

Zнаблюдаемое:

In [8]:
m0 = 17
m = 17.5
σ = np.sqrt(4)
n = 100

In [9]:
Z = (m - m0)/(σ/np.sqrt(n))
Z

2.5

Ответ: Zн > Zт, гипотеза H0 НЕ верна. Средний диаметр != 17мм

### 4) Продавец утверждает, что средний вес пачки печенья составляет 200 г. Из партии извлечена выборка из 10 пачек. Вес каждой пачки составляет: 202, 203, 199, 197, 195, 201, 200, 204, 194, 190. Известно, что их веса распределены нормально. Верно ли утверждение продавца, если учитывать, что доверительная вероятность равна 99%?

H0: w=200
H1: w != 200

Критерий: tтабличное = 3.2498

t наблюдаемое:

In [10]:
X = np.array([202, 203, 199, 197, 195, 201, 200, 204, 194, 190])

m0 = 200
m = X.mean()
σ = X.std(ddof=1)
n = 10

In [11]:
t = (m - m0)/(σ/np.sqrt(n))
t

-1.0651074037450896

Ответ: утверждение продавца верное, |t наблюдаемое| < tтабл.